packages/spacecat-shared-data-access/src/service/index.js

```diff
@@ -18,13 +18,16 @@ import { siteCandidateFunctions } from './site-candidates/index.js';
 import { organizationFunctions } from './organizations/index.js';
 import { configurationFunctions } from './configurations/index.js';
 import { siteTopPagesFunctions } from './site-top-pages/index.js';
+import { importJobFunctions } from './import-job/index.js';
+import { importUrlFunctions } from './import-url/index.js';
 
 /**
  * Creates a data access object.
  *
  * @param {{pkAllSites: string, pkAllLatestAudits: string, indexNameAllLatestAuditScores: string,
  * tableNameAudits: string,tableNameLatestAudits: string, indexNameAllSitesOrganizations: string,
  * tableNameSites: string, tableNameOrganizations: string, indexNameAllSites: string,
+ * tableNameImportJobs: string, pkAllImportJobs: string, indexNameAllImportJobs: string,
```

Comment:
Should there be a property(ies) added for the importUrl related bits as well?

Reply:
Currently we do not have any GSI's defined for importUrl. So, no properties have been added for the importUrl related bits.